Cargo.toml

```diff
@@ -60,4 +60,4 @@ uuid = "1.6.1"
 pem = "3.0.4"
 odbc-api = { version = "7.0.0" }
 arrow-odbc = { version = "9.0.0" }
-snowflake-api = { version = "0.8.0"}
+snowflake-api = { git="https://github.com/sgrebnov/snowflake-rs.git", folder="snowflake-api", branch="0.8.0-async-exec" }
```

Comment:
Let's add these commits to https://github.com/spiceai/snowflake-rs

Reply:
Updated - thank you